packages/frontend/src/types.ts

```diff
@@ -7,8 +7,25 @@ import type { SelectOption } from "@carrot-kpi/ui";
 import type { Dayjs } from "dayjs";
 import type { Address } from "viem";
 
+export enum ConstraintType {
```

Comment:
Should we move enum definition in the `sdk` lib? Since it could be used by all the different templates.

Reply:
It could be done... maybe for now I'd leave it here but it could be worth it to have an issue in the monorepo to remind us of this.